packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -89,6 +89,11 @@ export interface DashboardLayoutProps {
    * @default false
    */
   hideNavigation?: boolean;
+  /**
+   * Width of the sidebar when expanded, in pixels.
+   * @default 320
+   */
+  sidebarExpandedWidth?: number;
```

Comment:
Do we want (is it possible) to allow lengths in percent? or even a calculation such as `min(200px, 20%)`? we don't have to do this as part of this PR, it's just a question.

Reply:
My personal feedback is no. A fix width for the bar is more reasonable for design.

Reply:
I'll test if it works, if it does we can allow strings too, i guess it would be a simple change.